packages/@aws-cdk/aws-batch/README.md

```diff
@@ -13,39 +14,271 @@
 ---
 <!--END STABILITY BANNER-->
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+AWS Batch is a batch processing tool for efficiently running hundreds of thousands computing jobs in AWS. Batch can dynamically provision different types of compute resources based on your resource requirements for batch jobs submitted. Batch simplifies the planning, scheduling, and executions of your batch workloads across a full range of compute services like [Amazon EC2](https://aws.amazon.com/ec2/) and [Spot Resources](https://aws.amazon.com/ec2/spot/).
+
+Batch achieves this by utilizing queue processing of batch job requests. To successfully handle a job request, you need the following resources:
+
+1. [Job Definition](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-batch.JobDefinition.html) - *the container image to handle a job request*
```

Comment:
> This is hard to phrase since it's not just the container image. I imagine you singled it out since its the only one that cannot be overridden during job submission?

That was exactly the reasoning. Lets bring this change in then.